usaspending_api/broker/management/commands/load_fsrs.py

```diff
@@ -165,11 +167,9 @@ def gather_next_subawards(db_cursor, award_type, subaward_type, max_id, offset):
 
         query = "SELECT " + ",".join(query_columns) + " FROM fsrs_" + award_type + " AS award " + \
                 "JOIN fsrs_" + subaward_type + " AS sub_award ON sub_award.parent_id = award.id WHERE award.id > " + \
-                str(max_id) + "AND sub_award.subcontract_num IS NOT NULL ORDER BY award.id, sub_award.id LIMIT " + \
```

Comment:
I'm assuming sub_award.subcontract_num isn't required now?

Reply:
The query was having problems with saying `sub_award.subcontract_num IS NOT NULL`. It didn't think the column existed which was odd. Removing it fixed the query and that field looks to be populated on our end for all subawards.

Reply:
Confirming the need for this with @alburde1